python/paddle/incubate/distributed/utils/io/dist_save.py

```diff
@@ -47,43 +47,49 @@ def save(state_dict, path, **configs):
     Args:
         obj(Object) : The object to be saved.
         path(str|BytesIO) : The path/buffer of the object to be saved.
-          If saved in the current directory, the input path string will be used as the file name.
+            If saved in the current directory, the input path string will be used as the file name.
         protocol(int, optional): The protocol version of pickle module must be greater than 1 and less than 5.
-                                 Default: 4
+            Default: 4.
         **configs(dict, optional): optional keyword arguments. The following options are currently supported:
-          (1)use_binary_format(bool):
-            To be used in paddle.save. When the saved object is static graph variable, you can specify ``use_binary_for_var``.
-            If True, save the file in the c++ binary format when saving a single static graph variable; otherwise, save it in pickle format.
-            Default: False
-          (2)gather_to(int|list|tuple|None):
-            To specify which global rank to save in.Defalut is None.
-            None value means distributed saving with no gathering to a single card.
-          (3)state_type(str):
-            Value can be 'params' or 'opt', specifying to save parametres or optimizer state.
-          (4)max_grouped_size(str|int):
-            To limit the max size(how many bits) a object group to be transfered a time.
-            If str, the format must be as num+'G/M/K', for example, 3G, 2K, 10M, etc. Default is 3G.
+            1. use_binary_format(bool):
+                To be used in paddle.save. When the saved object is static graph variable, you can specify ``use_binary_for_var``.
+                If True, save the file in the c++ binary format when saving a single static graph variable; otherwise, save it in pickle format.
+                Default: False.
+            2. gather_to(int|list|tuple|None):
+                To specify which global rank to save in.Defalut is None.
+                None value means distributed saving with no gathering to a single card.
+            3. state_type(str):
+                Value can be 'params' or 'opt', specifying to save parametres or optimizer state.
+            4. max_grouped_size(str|int):
```

Comment:
```suggestion

            1. use_binary_format(bool):
                To be used in paddle.save. When the saved object is static graph variable, you can specify ``use_binary_for_var``.
                If True, save the file in the c++ binary format when saving a single static graph variable; otherwise, save it in pickle format.
                Default: False.
            2. gather_to(int|list|tuple|None):
                To specify which global rank to save in.Defalut is None.
                None value means distributed saving with no gathering to a single card.
            3. state_type(str):
                Value can be 'params' or 'opt', specifying to save parametres or optimizer state.
            4. max_grouped_size(str|int):
```